BaaServiceKit-Example/BaaServiceKit-Example/Modules/List/ListViewController.swift

```diff
@@ -30,33 +37,35 @@ class ViewController: UIViewController {
                 print(error)
             }
         }
-        
-        let record = Record(identifier: "1", description: self.randomString(length: 7))
+    }
+
+    func nodeExample() {
+        let record = Record(identifier: "1", description: String.random(length: 7))
         print(record)
-        
+
         // Create node example
         do {
-            
+
             // 1. Generate SHA-256
             let hashData = try self.blockchainService.generateSHA256(from: record)
-            
+
             // 2. Discover node URLs
             self.blockchainService.discoverPublicNodeURLs(completion: nil)
-            
+
             // 3. Submit hashes
             self.blockchainService.submit(hashes: [hashData], forNumberOfNodes: 1) {  [weak self] (result) in
-                
+
                 switch result {
                 case .success(let nodeResults):
-                    
+
                     // 4. Proof for submitted hashes
                     // Waiting to create the node on the service
-//                    delay(seconds: 20.0, completion: { [weak self] in
-//                        self?.proof(for: nodeResults)
-//                    })
+                    //                    delay(seconds: 20.0, completion: { [weak self] in
```

Comment:
FYI: the delay is needed because if the user submit and immediately create a proofRequest the response will be invalid.

Reply:
I will move this somewhere else, the two will not be embedded like this.